app/frontends/ogc-coverages/post-coverage-rangeset.ts

```diff
@@ -20,8 +20,8 @@ export default function postCoverageRangeset(
   res: Response,
   next: NextFunction,
 ): void {
-  // copy form parameters into the query
-  req.query = req.body;
+  // merge form parameters into the query
+  req.query = { ...req.query, ...req.body };
```

Comment:
This means that if the same key is passed in both as a query parameter and a form parameter we will use the value from the form. Does that make sense? Should we do any kind of validation, or log anything?

Reply:
Good point. So, if same key exists from both query and form parameter, which one should we use?

Reply:
I think if the same key is passed in both places with different values it is probably a mistake and should result in an error.